### PR TITLE
Sprint 22

### PR DESCRIPTION
**Pull request by ericsorenson**
## [1171 Petitions: QC - changes to QC section based upon Petitions Section feedback](https://github.com/flexion/ef-cms/issues/1171)
image

## [1936 Practitioner/IRS Attorney: Edit and display personal contact data](https://github.com/flexion/ef-cms/issues/1936)
image

## [2305 Clean-up: Signed Stipulated Decision](https://github.com/flexion/ef-cms/issues/2305)
image

### CODE REVIEW

**Comment by waldoj** [comment]
Here's the review checklist that I'll work my way through in reviewing this PR:

- [ ] begin by deleting any existing copy of the Git repository and cloning a fresh copy, running `docker system prune` to remove any cached artifacts from any prior code review, and running `./reset-dependencies.sh` to update Node packages
- [ ] review the pull request itself, to get oriented
	- [ ] read the description of the pull request, which should summarize the changes made
	- [ ] read through every task on the Scrum board that's encompassed by this pull request
	- [ ] read the description of the commits that comprise the pull request
- [ ] stand up the site locally
	- [ ] test all functionality in all major browsers, emphasizing the functionality that this pull request addresses
		- [ ] for internal Court functionality, perform the most thorough testing in Chrome, though also test in Edge and Firefox
		- [ ] for public-facing functionality, test in browsers consistent with [public browser use data](https://analytics.usa.gov/), including Internet Explorer 11
		- [ ] test in Mobile Safari and Mobile Chrome, with the caveat that not all internal Court functionality will be necessary on these platforms
	- [ ] use an automated audit tool for code quality and practices (recommended: [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/), aka [Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk))
		- [ ] look at efficiency of page loads, asset sizes, HTTP connection management, etc.
	- [ ] review for accessibility
		- [ ] use an automated audit tool, such as Chrome Audit or aXe
		- [ ] navigate site only with the keyboard
		- [ ] use VoiceOver or Narrator to navigate the site with audio only, with the display turned off
		- [ ] manually test anything that pa11y cannot test automatically (e.g., contrast of text over images)
- [ ] review static code analysis results in SonarCloud
- [ ] run a security audit of dependencies (`npm audit`) to ensure that there are no vulnerabilities that will be deployed to production (as opposed to vulnerabilities that only have an impact on the development environment)
- [ ] examine OWASP ZAP output in `docs/`, to ensure that any errors are known to be false positives or have been previously declared to be acceptable
- [ ] skim all new code, in the context of existing code, [looking for problems](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/CODE_REVIEW.md#what-we-look-for) (knowing that the vast majority of new code will be covered by tests)
- [ ] review all tests
	- [ ] look at code coverage of tests in SonarCloud
	- [ ] methodically review all new tests for correctness, quality of naming
- [ ] determine what code isn’t tested, review that rigorously
- [ ] review documentation to ensure that it matches changes
- [ ] provide comments on the pull request on GitHub, as necessary
	- [ ] for comments that are specific to a particular line of code, comment on those specific lines
	- [ ] for comments that are more general, attach the comment to a random line in `README.md` (as opposed to commenting on the pull request itself), to be able to use GitHub's ability to thread discussions on those comments
- [ ] for each feature-level bug (i.e., it’s working as designed, but designed wrong), open a new issue and put it in the backlog

**Comment by waldoj** [comment]
Note that `npm audit` reports:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ serialize-to-js                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.0.0                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ parcel-bundler [dev]                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ parcel-bundler > serialize-to-js                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/790                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

That is in the category of "accurate, but unhelpful." This is addressed within `package-lock.json`, but `npm audit` only looks at `package.json.` That is to say that this is a false positive, and I am ignoring it.

File: web-client/src/presenter/computeds/primaryContactEditHelper.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      },
+    };
+
+    const result = runCompute(primaryContactEditHelper, {
+      state: testState,
+    });
+    expect(result.showInCareOf).toBeTruthy();
+  });
+
+  it('should set showInCareOf to true when the caseDetail partType is estateWithoutExecutor', () => {
```
Comment (waldoj) [inline web-client/src/presenter/computeds/primaryContactEditHelper.test.js]:
```suggestion
  it('should set showInCareOf to true when the caseDetail partyType is estateWithoutExecutor', () => {
```

File: policy.json.tpl [hunk 2]
```diff
@@ -159,6 +160,9 @@
       "Resource": [
         "arn:aws:iam::ACCOUNT_ID:role/api_gateway_cloudwatch_global_*",
         "arn:aws:iam::ACCOUNT_ID:role/ef-cms-cases-*-lambdaRole",
+        "arn:aws:iam::ACCOUNT_ID:role/ef-cms-case-documents-*-lambdaRole",
```
[the remark this answers is not in the document]
Comment (sutt0n) [reply policy.json.tpl]: Correct @waldoj

**Pull request merged**